polaris-react/src/components/Navigation/Navigation.scss

```diff
@@ -203,9 +177,42 @@ $disabled-fade: 0.6;
 }
 
 .ItemWrapper {
+  width: 100%;
+  padding: 0 var(--p-space-2);
+}
+
+.ItemInnerWrapper {
   display: flex;
   flex-wrap: nowrap;
   width: 100%;
+
+  &:hover {
+    background: var(--p-background-hovered);
+    color: var(--p-text);
+    text-decoration: none;
+  }
+}
+
+.ItemInnerWrapper-Selected {
+  background-color: var(--p-background-selected);
+  @include focus-ring;
+
+  &:hover,
+  &.keyFocused {
+    background-color: var(--p-background-hovered);
+    color: var(--p-text-primary-hovered);
+  }
+
+  .keyFocused {
+    @include focus-ring($style: 'focused');
+  }
+
+  &:active,
```

Comment:
I don't think `:active` would work on a wrapping `div`. This is probably a no-op.

Reply:
I think we'd need move the active rules back into the `a`.

Reply:
Yup confirmed. The text turns black but should stay green:
![green](https://user-images.githubusercontent.com/1307190/204581287-b53d1336-708e-4683-a87b-122c836d770c.gif)